src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers/JavaCallableWrapperGenerator.cs

```diff
@@ -387,6 +387,12 @@ void AddConstructor (MethodDefinition ctor, TypeDefinition type, string? outerTy
 				} else if (v.Name == "GenerateJavaPeer") {
 					r.DoNotGenerateAcw = ! (bool) v.Argument.Value;
 				}
+				var isKeyProp   = attr.Properties.FirstOrDefault (p => p.Name == "IsKeyword");
+				var isKeyword	= isKeyProp.Name != null && ((bool) isKeyProp.Argument.Value) == true;
+				var arrRankProp = attr.Properties.FirstOrDefault (p => p.Name == "ArrayRank");
```

Comment:
Is the C# attribute available where we could use `nameof()`? The resulting IL would be the same, but might help typos.

Reply:
No, this assembly doesn't have a reference to `Java.Interop.dll`.